assets/css/_variables.scss

```diff
@@ -1,5 +1,6 @@
 $font-size: (
   sm: .875rem,   // 14px
+  md: 1rem,   // 14px
```

Comment:
```suggestion
  md: 1rem,   // 16px
```

Reply:
@rosle Fixed issues in [6c62c24](https://github.com/rafayet-monon/elixir_search_extractor/pull/30/commits/6c62c2457e71b799e3948236c09ce4a9fefa47fe)